addons/mail/wizard/mail_resend_message_views.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="mail_resend_message_view_form" model="ir.ui.view">
+            <field name="name">mail.resend.message.form</field>
+            <field name="model">mail.resend.message</field>
+            <field name="groups_id" eval="[(4,ref('base.group_user'))]"/>
+            <field name="arch" type="xml">
+                <form string="Edit Partners">
+                    <field name="mail_message_id" invisible="1"/>
+                    <field name="notification_ids" invisible="1"/>
+                    <field name="has_cancell" invisible="1"/>
+                    <p>Please, select the action to do on each email and correct the email if needed. The modified email will be saved on partner</p>

```

Comment:
_Please, select the action to do on each mail and correct the email address if needed. The modified address will be saved on the corresponding contact._